File: server/models/label.js
const { DataTypes } = require('sequelize');

module.exports = sequelize => {
  sequelize.define('label', {
    title: {
      allowNull: false,
      type: DataTypes.STRING,
    },
    description: {
      type: DataTypes.STRING,
    },
    color: {
      allowNull: false,
      type: DataTypes.STRING,
    },
  });
};

File: client/iOS/IssueTracker/IssueTracker/Model/Label.swift
//
//  Label.swift
//  IssueTracker
//
//  Created by A on 2020/11/02.
//

import Foundation

struct Label: Codable, Hashable, Badgeable, Identifiable {
    let id: Int?
    let title: String
    let description: String?
    let color: String
}
